ecommerce/serializers.py

```diff
@@ -258,6 +259,11 @@ def get_amount(self, instance):
         # decimal fields should be represented as strings to prevent floating point parsing problems
         return str(latest_coupon_version(instance.coupon).payment_version.amount)
 
+    def get_discount_type(self, instance):
+        """ Get the coupon discount amount """
+        # decimal fields should be represented as strings to prevent floating point parsing problems
+        return str(latest_coupon_version(instance.coupon).payment_version.discount_type)
```

Comment:
Any specific reason for calling str?

Reply:
I think we can remove that conversion, I'll change that in the next commit.

Reply:
Addressed in https://github.com/mitodl/mitxpro/pull/2455/commits/57ef368881d053254364742e56e270d4dad67fc6